yrs/src/block.rs

```diff
@@ -156,7 +156,7 @@ impl BlockPtr {
                         .blocks
                         .get_item_clean_end(&ID::new(this.id.client, this.id.clock - 1));
                     this.origin = this.left.as_deref().map(|b: &Block| b.last_id());
-                    this.content.splice(offset as usize);
+                    this.content = this.content.splice(offset as usize).unwrap();
```

Comment:
Fixed one bug, where resending the merged block (which beginning was seen in the past) was causing a content splicing, where wrong half of spliced data was being attached.